doc/source/index.rst

```diff
@@ -4,74 +4,59 @@
-
-  Multiple threads on slave performing replication on row level.
-
-* **Automatic node provisioning**
-
-  You simply add a node and it automatically syncs.
-
-* **Data consistency**
-
-  No more unsynchronized nodes.
-
-* **PXC Strict Mode**
-
-  Avoids the use of experimental and unsupported features.
-
-* **Configuration script for ProxySQL**
-
-  Percona provides a ProxySQL package with the ``proxysql-admin`` tool
-  that automatically configures |PXC| nodes.
-
-* **Automatic configuration of SSL encryption**
-
-  |PXC| includes the ``pxc-encrypt-cluster-traffic`` variable
-  that enables automatic configuration of SSL encrytion.
-
-* **Optimized Performance**
-
-  |PXC| performance is optimized to scale with a growing production workload.
-
-  For more information, see the following blog posts:
-
-  * `How We Made Percona XtraDB Cluster Scale
-    <https://www.percona.com/blog/2017/04/19/how-we-made-percona-xtradb-cluster-scale/>`_
-
-  * `Performance improvements in Percona XtraDB Cluster 5.7.17-29.20
-    <https://www.percona.com/blog/2017/04/19/performance-improvements-percona-xtradb-cluster-5-7-17/>`_
-
-|PXC| is fully compatible with `MySQL Server Community Edition <MySQL>`_,
+|PXC|_ is a database clustering solution for MySQL.  It ensures high
+availability, prevents downtime and data loss, and provides linear scalability
+for a growing environment.
+
+.. rubric:: Features of |PXC|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - Details
+   * - Synchronous replication**
+     - Data is written to all nodes simultaneously, or not written at all in
+       case of a failure even on a single node
+   * - Multi-master replication
+     - Any node can trigger a data update.
+   * - True parallel replication
+     - Multiple threads on slave performing replication on row level
+   * - Automatic node provisioning
+     - You simply add a node and it automatically syncs.
+   * - Data consistency
+     - No more unsynchronized nodes.
+   * - PXC Strict Mode
+     - Avoids the use of experimental and unsupported features
+   * - Configuration script for ProxySQL
+     - |PXC| includes the ``proxysql-admin`` tool that automatically configures
+       |PXC| nodes using ProxySQL.
+   * - Automatic configuration of SSL encryption
+     - |PXC| includes the ``pxc-encrypt-cluster-traffic`` variable
+       that enables automatic configuration of SSL encrytion
+   * - Optimized Performance
+     - |PXC| performance is optimized to scale with a growing production
+       workload
+
+.. seealso::
+   How We Made Percona XtraDB Cluster Scale
+       https://www.percona.com/blog/2017/04/19/how-we-made-percona-xtradb-cluster-scale
+   Performance improvements in Percona XtraDB Cluster 5.7.17-29.20
+       https://www.percona.com/blog/2017/04/19/performance-improvements-percona-xtradb-cluster-5-7-17
+
+|PXC| |version| is fully compatible with `MySQL Server Community Edition <MySQL>`_,
 |PS|_, and MariaDB_ in the following sense:
 
```

Comment:
We should mention what MySQL/PS versions are compatible.
What do we mean when we say with MariaDB? Latest MariaDB versions come with integrated galera, do we support PXC<->MariaDB clusters?